flixel/text/FlxText.hx

```diff
@@ -140,6 +140,7 @@ class FlxText extends FlxSprite
 	var _defaultFormat:TextFormat;
 	var _formatAdjusted:TextFormat;
 	var _formatRanges:Array<FlxTextFormatRange> = [];
+	var _toRemoveFormatRanges:Array<FlxTextFormatRange> = [];
```

Comment:
This should be a local variable, not a field. But there's an even better solution: iterate backwards over `_formatRanges` using a `while` loop, then it's safe to remove while iterating.

Reply:
I pushed a new commit with these changes, thanks!
(note: I found a bug in how openfl handles text formatting in TextField for neko, so will be following up there).